src/containers/audio-rewards-page/WalletModal.module.css

```diff
@@ -4,6 +4,14 @@
   width: 100%;
 }
 
+.modalBody.wallets {
```

Comment:
why '.modalBody.wallets' twice?

Reply:
![Screen Shot 2021-03-25 at 3 53 00 PM](https://user-images.githubusercontent.com/7064122/112535056-32a6f700-8d82-11eb-9e2e-6458850d2831.png)
I wanted this selector to be more specific if the modal body was displaying wallets to modify the width

Reply:
Oh, I misread the comment, yeah will merge into one